GPU/Common/TextureDecoder.cpp

```diff
@@ -610,10 +626,11 @@ uint32_t GetDXT5Texel(const DXT5Block *src, int x, int y) {
 }
 
 // This could probably be done faster by decoding two or four blocks at a time with SSE/NEON.
-void DecodeDXT1Block(u32 *dst, const DXT1Block *src, int pitch, int height, bool ignore1bitAlpha) {
+void DecodeDXT1Block(u32 *dst, const DXT1Block *src, int pitch, int height, u32 *alpha) {
 	DXTDecoder dxt;
-	dxt.DecodeColors(src, ignore1bitAlpha);
+	dxt.DecodeColors(src, false);
 	dxt.WriteColorsDXT1(dst, src, pitch, height);
+	*alpha = dxt.AnyNonFullAlpha() ? 0 : 1;
```

Comment:
nit: I realize we set the one bit, but seems like `-1` or `0xFFFFFFFF` might be clearer here.

-[Unknown]